invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -60,12 +60,19 @@ <h1>{{ metadata.title }}</h1>
         {%- endblock record_content -%}
         {# files #}
         {%- block record_files -%}
-        {%- if record.files and record|can_list_files %}
-          {%- set files = record.files|make_files_preview_compatible %}
-          {%- set default_preview_file = files|select_preview_file(default_preview=record.files.default_preview) %}
-          {{ preview_file_box(default_preview_file, pid) }}
-          {{ file_list_box(files, pid) }}
-        {% endif %}
+        {%- if files is not none -%}
+          {%- if files.enabled and record|can_list_files %}
+            {%- set files = files|order_entries %}
+            {%- if files|has_previewable_files -%}
+              {%- set default_preview_file = files|select_preview_file(default_preview=files.default_preview) %}
+              {{ preview_file_box(default_preview_file, pid) }}
+            {%- endif -%}
+            {{ file_list_box(files, pid) }}
+          {% endif %}
+        {%- else -%}
+          {# TODO: come up with a proper display #}
+          <div>Sorry, the files are restricted!</div>
```

Comment:
should the message be included here, or rather in the view via `flash`ing messages?

Reply:
This is part of #595 to have a nice page so we can simply integrate like this. Ping @zzacharo